datasimulator/main.py

```diff
@@ -144,9 +143,13 @@ def run_submission_order_generation(graph, data_path, node_name=None):
     else:
         submission_order = graph.generate_submission_order()
 
-    with open(os.path.join(data_path, "DataImportOrderPath.txt"), "w") as outfile:
-        for node in submission_order:
-            outfile.write(node.name + "\t" + node.category + "\n")
+    file_path = os.path.join(data_path, "DataImportOrderPath.txt")
```

Comment:
In the PR description:
> New Features
When running data-simulator command that create this file, it no longer will crash with a misleading message about the .txt file

What is "this file"? This will end up in release notes and needs to make sense without opening the PR

Reply:
DataImportOrderPath.txt

Reply:
The note should also mention under what circumstances it crashed, right now it sounds like it was always crashing

Keep it simple and non-technical, eg "Improve error log when `data-simulator submission_order --path` cannot find the specified path"

Reply:
note should hopefully be clearer now